tests/test_client_workitems.py

```diff
@@ -239,6 +244,30 @@ def test_create_work_items_successfully(
     assert json.loads(req.content.decode()) == expected
 
 
+def test_create_work_item_in_document(
+    client: polarion_api.OpenAPIPolarionProjectClient,
+    httpx_mock: pytest_httpx.HTTPXMock,
+    work_item: polarion_api.WorkItem,
+):
+    with open(TEST_WI_CREATED_RESPONSE, encoding="utf8") as f:
+        mock_response = json.load(f)
+
+    httpx_mock.add_response(201, json=mock_response)
+
+    work_item.home_document = polarion_api.DocumentReference(
+        "space", "document"
+    )
+    client.create_work_items([work_item])
+
+    req = httpx_mock.get_request()
+
+    assert req is not None and req.method == "POST"
+    with open(TEST_WI_MULTI_POST_REQUEST_IN_DOC, encoding="utf8") as f:
+        expected = json.load(f)
+
+    assert json.loads(req.content.decode()) == expected
```

Comment:
Keep the 1. prepare, 2. act and 3. assert structure of tests. Additionally I am more used to the `.read_text()` method way of `pathlib.Path`s:

```suggestion
    mock_response = json.load(
        TEST_WI_CREATED_RESPONSE.read_text(encoding="utf8")
    )
    expected = json.load(
        TEST_WI_MULTI_POST_REQUEST_IN_DOC.read_text(encoding="utf8")
    )
    httpx_mock.add_response(201, json=mock_response)
    work_item.home_document = polarion_api.DocumentReference(
        "space", "document"
    )

    client.create_work_items([work_item])

    assert (req := httpx_mock.get_request()) and req.method == "POST"
    assert json.loads(req.content.decode()) == expected
```